models/dex/dex_prices.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(
-    alias = 'prices',
+    alias = alias('prices'),
```

Comment:
at top of this config block, add:
`tags=['dunesql'],`

Reply:
added on [39ed84f](https://github.com/duneanalytics/spellbook/pull/3684/commits/39ed84f0f28fcb768756a617a4aad760f2f5b739)